cpp/ql/lib/semmle/code/cpp/dataflow/ExternalFlow.qll

```diff
@@ -138,6 +139,9 @@ predicate sourceModel(
     row.splitAt(";", 7) = kind
   ) and
   provenance = "manual"
+  or
+  Extensions::sourceModel(namespace, type, subtypes, name, signature, ext, output, kind, provenance,
+    _)
```

Comment:
Once we've merged this we should probably create an issue for the final part of this issue: Supporting provenance. This should be done by replacing the `_` with a `QlBuiltins::ExtensionId` parameter. But I think delaying this is perfectly fine!

Reply:
Yep, there's already an internal issue for this.